### ISSUE TITLE
[Bug]: Direct links to charts are not working anymore

### ISSUE THREAD

**Issue by ekexcello** [netdata/netdata#11984]
### Bug description

Previously it was handy to copy a link to particular chart when it's zoomed to certain time range. Now these links are not working anymore, instead, a dashboard and latest state is displayed by Netdata.

For example, say, there's a network traffic peak happened one day ago. Until 1.32 it was possible to find an interesting chart (for example, `http://server3.domain.net:19999/#after=1642215003000;before=1642227213000;menu_ipv6_submenu_tcp6=undefined;theme=slate` and share this chart wit colleagues. On opening the link they could see the very chart and time range when traffic peak happened.
Now, instead of this chart, a default dashboard opens (just like you simply open `http://server3.domain.net:19999/`)

### Expected behavior

On opening the link pointing explicitly to the chart by name and by time range, that very chart should be displayed zoomed to the time interval provided with the same link.

### Steps to reproduce

1. Open netdata interface on any server
2. Scroll to preferred chart, zoom to desired time range
3. Copy the link from address bar and open it with another browser

### Installation method

other

### System info

```shell
Linux 5.10.91-gentoo #1 SMP Tue Jan 11 23:57:41 CET 2022 x86_64 AMD EPYC 7451 24-Core Processor AuthenticAMD GNU/Linux
/etc/gentoo-release:Gentoo Base System release 2.7
/etc/os-release:NAME=Gentoo
/etc/os-release:ID=gentoo
/etc/os-release:PRETTY_NAME="Gentoo/Linux"
/etc/os-release:ANSI_COLOR="1;32"

Linux 4.19.0-18-amd64 #1 SMP Debian 4.19.208-1 (2021-09-29) x86_64 GNU/Linux
/etc/os-release:PRETTY_NAME="Debian GNU/Linux 10 (buster)"
/etc/os-release:NAME="Debian GNU/Linux"
/etc/os-release:VERSION_ID="10"
/etc/os-release:VERSION="10 (buster)"
/etc/os-release:VERSION_CODENAME=buster
/etc/os-release:ID=debian
```

### Netdata build info

```shell
Version: netdata v1.32.1-78-nightly
Configure options:  '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var' '--libexecdir=/usr/libexec' '--libdir=/usr/lib' '--with-zlib' '--with-math' '--with-user=netdata' '--disable-ml' '--with-bundled-protobuf' 'CFLAGS=-O2' 'LDFLAGS='
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES 
    ACLK Next Generation:       YES
    ACLK-NG New Cloud Protocol: YES
    ACLK Legacy:                NO
    TLS Host Verification:      YES
    Machine Learning:           NO
Libraries:
    protobuf:                NO
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    NO
    EBPF:                    YES
    IPMI:                    NO
    NFACCT:                  NO
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: NO

Version: netdata v1.32.1
Configure options:  '--prefix=/usr' '--build=x86_64-pc-linux-gnu' '--host=x86_64-pc-linux-gnu' '--mandir=/usr/share/man' '--infodir=/usr/share/info' '--datadir=/usr/share' '--sysconfdir=/etc' '--localstatedir=/var/lib' '--disable-dependency-tracking' '--disable-silent-rules' '--docdir=/usr/share/doc/netdata-1.32.1' '--htmldir=/usr/share/doc/netdata-1.32.1/html' '--libdir=/usr/lib64' '--localstatedir=/var' '--with-user=netdata' '--disable-cloud' '--without-aclk-ng' '--enable-jsonc' '--disable-plugin-cups' '--enable-dbengine' '--enable-plugin-nfacct' '--disable-plugin-freeipmi' '--disable-backend-kinesis' '--enable-lto' '--disable-backend-mongodb' '--disable-backend-prometheus-remote-write' '--disable-plugin-xenstat' '--enable-x86-sse' '--with-zlib' 'build_alias=x86_64-pc-linux-gnu' 'host_alias=x86_64-pc-linux-gnu' 'CFLAGS=-march=native -O2 -pipe' 'LDFLAGS=-Wl,-O1 -Wl,--as-needed' 'CXXFLAGS=-march=native -O2 -pipe' 'PKG_CONFIG_PATH=/var/tmp/portage/net-analyzer/netdata-1.32.1/temp/python3.9/pkgconfig'
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              NO (by user request)
    ACLK Next Generation:       NO
    ACLK-NG New Cloud Protocol: NO
    ACLK Legacy:                NO
    TLS Host Verification:      YES
    Machine Learning:           YES
Libraries:
    protobuf:                NO
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  YES
    libcrypto:               YES
    libm:                    YES
    LWS:                     YES shared-lib
    mosquitto:               YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    NO
    EBPF:                    NO
    IPMI:                    NO
    NFACCT:                  YES
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: NO
```

### Additional info

Two systems tested.
Installation method on Gentoo is gentoo package (from source)
Installation on Debian was done with kickstart.sh
Same behavior can be seen regardless of browser used (Firefox 96.0, Google Chrome 97.0.4692.71 Official 64-bit are tested, same issue reported by others with various browser versions used)

**Comment by ilyam8** [comment]
Hi, @ekexcello. We confirm this issue. The [bug report](https://github.com/netdata/dashboard/issues/346) is open in netdata/dashboard.

**Comment by ilyam8** [comment]
Closing in favor of netdata/dashboard#346. That is the dashboard bug, was introduced in [v2.20.7](https://github.com/netdata/netdata/pull/11629).

**Comment by ilyam8** [comment]
Fixed in #12052
